client/src/pages/reports/Compact.js

```diff
@@ -0,0 +1,939 @@
+      report.attendees.filter(attendee => {
+        // filter advisor attendees which are not the primary one we added above
+        return (
+          attendee.role === Person.ROLE.ADVISOR &&
+          attendee.uuid !== advisorAttendees[0]?.uuid
+        )
+      })
+    )
+    return getAttendeesAndAssessments(sortGroupByOrganization(advisorAttendees))
+  }
+
+  function getTasksAndAssessments() {
+    return (
+      <table>
+        <tbody>
+          {report.tasks.map(task => {
+            const taskInstantAssessmentConfig = task.getInstantAssessmentConfig()
+            // return only name and objective if no assessment
+            return (
+              <CompactRow
+                key={task.uuid}
+                label={<LinkTo modelType={Task.resourceName} model={task} />}
+                content={
+                  <table>
+                    <tbody>
+                      <CompactRow
+                        label={Settings.fields.task.topLevel.shortLabel}
+                        content={
+                          task.customFieldRef1 && (
+                            <LinkTo
+                              modelType="Task"
+                              model={task.customFieldRef1}
+                            >
+                              {task.customFieldRef1.shortName}
+                            </LinkTo>
+                          )
+                        }
+                      />
+                      {optionalFields.assessments.active &&
+                        taskInstantAssessmentConfig && (
+                          <ReadonlyCustomFields
+                            parentFieldName={`${Report.TASKS_ASSESSMENTS_PARENT_FIELD}.${task.uuid}`}
+                            fieldsConfig={taskInstantAssessmentConfig}
+                            values={report}
+                            vertical
+                            isCompact
+                          />
+                      )}
+                    </tbody>
+                  </table>
+                }
+              />
+            )
+          })}
+        </tbody>
+      </table>
+    )
+  }
+
+  function getAttendeesAndAssessments(attendees) {
+    // to keep track of different organization, if it is same consecutively, don't show for compactness
+    let prevDiffOrgName = ""
+    return (
+      <table>
+        <tbody>
+          {attendees.map(attendee => {
+            const attendeeInstantAssessmentConfig = attendee.getInstantAssessmentConfig()
+            const renderOrgName =
+              prevDiffOrgName !== attendee.position?.organization?.shortName
+            prevDiffOrgName = renderOrgName
+              ? attendee.position?.organization?.shortName
+              : prevDiffOrgName
+            return (
+              <CompactRow
+                key={attendee.uuid}
+                label={
+                  <>
+                    <LinkTo modelType="Person" model={attendee} />
+                    {renderOrgName && (
```

Comment:
It would be better to display ` 'NA'` for all people without an organization. Such that, `{(renderOrgName || !attendee.position?.organization) && (`